JS/fluentRevealNavbar.uc.js

```diff
@@ -149,7 +184,9 @@
         area = el.querySelector(".toolbarbutton-text");
       }
 
-      if (el.disabled || areaStyle.pointerEvents == "none") {
+      // previous pointerEvents check may break the effect in FF 115
+      //if (el.disabled || areaStyle.pointerEvents == "none") {
+      if (el.disabled) {
         return this.clearEffect(area);
       }
 
```

Comment:
The code below needs to be fixed, or there is a bug when you move the mouse around the top of the page on about:config or about:preferences, etc. It's because they are in the parent process so their true event targets pass all the way up to the chrome window listener (normally the event target we see would be `<browser>`).

It's better if we just use MousePosTracker.
```js
      let x = MousePosTracker._x - this.getOffset(area).left - window.scrollX;
      let y = MousePosTracker._y - this.getOffset(area).top - window.scrollY;
```

Reply:
Sorry, I didn't know about this bug, so my first thought was this check was added just for UX convenience. So now this should be handled by new `getComputedStyle(el).pointerEvents == "none"` check, right?